examples/use-cases/src/encrypt.ts

```diff
@@ -0,0 +1,43 @@
+import { VaultClient } from "@piiano/vault-client";
+
+async function main() {
```

Comment:
We can avoid the wrapping `main` function.
modern js now has top level `await` so for scripts like that we can write the code without a function.

```diff
- async function main() {
  // code
- }}
-
- main()
```

Reply:
I can't manage to make it happen in the combination of tsconfig/package.json and the various target options. Let's do it together @ori-shalom